### PR TITLE
#12-1 list_info page PR8

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (Checkpoint)
It is confirmation of this PR, list & information page function (information layout).

I fixed the problem.
<https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing> <Back Log # 12>

# # To test this PR
--- (If an event is stored in DB, from step 9) ---
1. Go to the login page (example: localhost: 8086 / login).
2. Input user credentials
3. Click Login
4. my_profile page
5. Click the Create Event button
6. Enter all content
7. Press the "Submit" button
8. (Go to the event_list page)
9. Top page (Ex: localhost: 8086)
10. Press the Event list button near the upper right
11. (Go to Event_list page)
12. Press the "Go Event" button on the displayed event card
13. (Go to Event_info page)

**command**
None
 **screenshot**
-<https://drive.google.com/drive/folders/1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W?usp=sharing>

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -3,6 +3,10 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+<<<<<<< Updated upstream
+=======
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Again. Please do not push conflicts.

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Again. Please do not push conflicts.

File: app/Http/Controllers/HomeController.php [hunk 3]
```diff
@@ -34,22 +34,23 @@ public function create()
...
-    //     $event->event_info = $request->input('event_info');
-    //     $event->image = 'image';
-    //     $event->save();
-    //     //Auth::user()->events()->save($event);
-    //     return redirect('/my_profile');
-    // }
+
+    public function post(Request $request)
+    {
+        // Auth::user()->events()->create()
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: Again, remove if you do not need anymore.

File: app/Http/Controllers/HomeController.php [hunk 4]
```diff
@@ -34,22 +34,23 @@ public function create()
...
+        $event->user_id = Auth::user()->id;
+        $event->event_title = $request->input('event_title');
+        $event->representative_name = $request->input('representative_name');
+        $event->date = $request->input('event_date');
+        $event->area = $request->input('area');
+        $event->category = $request->input('Category');
+        $event->event_info = $request->input('event_info');
+        $event->image = 'image';
+        $event->save();
+        //Auth::user()->events()->save($event);
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: Again, remove if you do not need anymore.

File: app/Http/Controllers/HomeController.php [hunk 5]
```diff
@@ -34,22 +34,23 @@ public function create()
...
-    //     $event->save();
-    //     //Auth::user()->events()->save($event);
-    //     return redirect('/my_profile');
-    // }
+
+    public function post(Request $request)
+    {
+        // Auth::user()->events()->create()
+        $event = new Event;
+        // dd($event);
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: Do not push dd(), it's only used for debugging.

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (kokioba95) [reply app/Http/Controllers/UserController.php]: I overlooked it.
Fixed.

File: app/Http/Controllers/UserController.php (hunk 2 above)
Comment (kokioba95) [reply app/Http/Controllers/UserController.php]: I overlooked it.
Fixed.

File: app/Http/Controllers/HomeController.php (hunk 3 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: I overlooked it.
its Delete.

File: app/Http/Controllers/HomeController.php (hunk 4 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: its Delete.

File: app/Http/Controllers/HomeController.php (hunk 5 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: its Delete.

**Pull request merged**